test/correctness/simd_op_check.cpp

```diff
@@ -106,7 +107,7 @@ struct Test {
         const bool can_run = can_run_code();
         for (auto p : image_params) {
             p.set_host_alignment(128);
-            p.dim(0).set_min(0);
+            p.dim(0).set_min(-PAD).set_extent(W + 2 * PAD);
             if (can_run) {
                 // Make a buffer filled with noise to use as a sample input.
                 Buffer<> b(p.type(), {W*4+H, H});
```

Comment:
I missed this with my padding change, you'll also need to set the min of this buffer to match the requirements of the param (dim(0) padding = -PAD). To verify this path is working, it would be a good idea to just run make correctness_simd_op_check (i.e. for the host/x86 target).